fhd_core/beam_modeling/fhd_struct_init_antenna.pro

```diff
@@ -119,59 +124,20 @@ undefine, xvals_celestial, yvals_celestial
 valid_i=where(Finite(ra_arr),n_valid)
 ra_use=ra_arr[valid_i]
 dec_use=dec_arr[valid_i]
-if ~keyword_set(beam_per_baseline) then undefine, dec_arr, ra_arr
```

Comment:
This code block through 161 should not be removed. It is necessary for reducing mem costs

Reply:
added back in